### PR TITLE
fix: Improve Kotlin Code

### LINKED ISSUE #1389: Improve Kotlin Code in SettingsFragment
**Describe the chore**
In settings Fragment, there are unnecessary usage of "context.let", which can be replaced by requireContext().
@iamareebjamal suggested this once.

**Would you like to work on the issue?**

- [x] Yes

### PR DESCRIPTION
**Pull request by somenath1435**
Fixes #1389 

Changes: Replaced context.let with requireContext() in SettingsFragment

### CODE REVIEW

**Comment by iamareebjamal** [comment]
Make the change everywhere in the project

**Comment by somenath1435** [comment]
@iamareebjamal Done.
Android studio has done some autoformat in code(Indentation changes also)

File: app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewHolder.kt [hunk 1]
```diff
@@ -28,14 +28,14 @@ class AttendeeViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
...
-                        ticket = TicketId(attendeeRecyclerAdapter.ticketList[position].id.toLong()),
-                        event = attendeeRecyclerAdapter.eventId)
+                    lastname = itemView.attendeeItemLastName.text.toString(),
+                    email = itemView.attendeeItemEmail.text.toString(),
+                    city = getAttendeeField("city"),
+                    address = getAttendeeField("address"),
+                    state = getAttendeeField("state"),
+                    country = itemView.attendeeItemCountry.text.toString(),
+                    ticket = TicketId(attendeeRecyclerAdapter.ticketList[position].id.toLong()),
+                    event = attendeeRecyclerAdapter.eventId)
```
Comment (iamareebjamal) [inline app/src/main/java/org/fossasia/openevent/general/attendees/AttendeeViewHolder.kt]: Revert

**Comment by iamareebjamal** [comment]
Revert unneeded changes

**Comment by somenath1435** [comment]
@iamareebjamal Done

**Comment by iamareebjamal** [comment]
Not done

**Pull request merged**